index.js

```diff
@@ -66,8 +66,8 @@ function processLength(/* CODE HERE */) {
  * Invoking `processLastItem` passing `['foo', 'bar']` and `(str) => str + str`,
  * should return 'barbar'.
 */
-function processLastItem(/* CODE HERE */) {
-  /* CODE HERE */
+function processLastItem(stringList, callback) {
+  return callback(stringList[1]);
 }
```

Comment:
This function should be returning the Last Item.  Right now it is returning the item from index 1.  Remember that we us arr.length-1 in replace of the index so that we are always reaching the last item.